cypress/support/fragments/settings/tenant/location-setup/libraries.js

```diff
@@ -1,6 +1,16 @@
 import uuid from 'uuid';
 import TenantPane, { getDefaultTenant } from '../baseTenantPane';
-import { EditableListRow, MultiColumnListCell, Link } from '../../../../../../interactors';
+import {
+  EditableListRow,
+  MultiColumnListCell,
+  Link,
+  Modal,
+  Button,
+  including,
+} from '../../../../../../interactors';
+
+const deleteModal = Modal({ id: 'delete-controlled-vocab-entry-confirmation' });
```

Comment:
I recommend to create separate modules for each Modal windows and encapsulate all related modules, like example:
cypress/support/fragments/orders/modals/selectLocationModal.js

Example of usage:
```JS
      // Click "Create new holdings for location" link in "Location" accordion
      const SelectLocationModal = OrderLines.openCreateHoldingForLocation();
      SelectLocationModal.verifyModalView();
```

Reply:
done